code/controllers/subsystem/ships.dm

```diff
@@ -119,7 +119,7 @@ SUBSYSTEM_DEF(ship)
 	var/starting_shields = S.shield_strength
 	if(world.time > S.next_recharge && S.recharge_rate)
 		S.next_recharge = world.time + S.recharge_rate
-		S.shield_strength = min(initial(S.shield_strength), S.shield_strength + 1)
+		S.shield_strength = min(initial(S.shield_strength), S.shield_strength + rand(750,1250))
```

Comment:
750 - 1250 is a stupid amount and would be OP on some ships, make this a ship var or use components in conjunction with this instead of a magic number

Reply:
750-1250 is the same as what it was, just with a little random variance. It used to just regen 1 point every repair phase.

Also not really OP since most ships are killed before even getting a repair tick due to OP cells.

Reply:
Yeah but were removing OP cells, we should code for the future